_episodes/01-introduction.md

```diff
@@ -73,18 +73,68 @@ on your computer. If you're using Windows, you'll need to download a separate pr
 ## Starting with the shell
 
 We will spend most of our time learning about the basics of the shell
-by manipulating some experimental data.
+by manipulating some experimental data. 
 
-First, we're going to download the data we will be working with. 
-For this you'll need internet access, because you're going to get it off the web.  
+Some of the data we're going to be working with is quite large, and
+we're also going to be using several bioinformatics packages in later
+lessons to work with this data. To avoid having to spend time 
+downloading the data and downloading and installing all of the software,
+we're going to be working with data on a remote server. 
 
-We're going to be working with data on our remote server. 
-
-You can log-in to the remote server using the information in the Etherpad. First 
+You can log-in to the remote server using the information that your instructor has pasted into the Etherpad. First 
 open your shell (through the Terminal application) and then enter the text starting
 with `ssh` from the Etherpad. Each of you will have a different log-in. This will 
 prevent us from accidentally changing each other's files as we work through the
-exercises.  
+exercises. For example, you might type:
+
+~~~
+$ ssh dcuser@ec2-34-229-196-68.compute-1.amazonaws.com
+~~~
+{: .bash}
+
+Make sure you replace `ec2-34-229-196-68.compute-1.amazonaws.com` with your individual log-in. 
+
+You will be prompted to enter a password. The password will be provided to you by your instructor.
+
+After loging on, you will see a screen something like this: 
```

Comment:
I would suggest: 
After logging on, you will see a screen showing something like this: